src/monstergenerator.cpp

```diff
@@ -454,7 +456,15 @@ void MonsterGenerator::load_monster(JsonObject &jo)
         newmon->dies = get_death_functions(jo, "death_function");
         load_special_defense(newmon, jo, "special_when_hit");
         load_special_attacks(newmon, jo, "special_attacks");
+
         newmon->upgrade_min = jo.get_int("upgrade_min", -1);
+        // recalc upgrade_min if given
+        if (newmon->upgrade_min > 0) {
+            const int season_scale = ACTIVE_WORLD_OPTIONS["SEASON_LENGTH"] / 14;
```

Comment:
This should not scale with season length, if someone wants longer seasons that does not imply they want slower monster progression or anything else, just longer seasons.  Weather is tied to seasons, plant growth is tied to seasons, animal reproduction is tied to seasons, nothing else should be.